### PR TITLE
test undercloud nodes resources

### PR DESCRIPTION
**Pull request by rrasouli**
This test purpose checking openstack infra provider hosts CPU and Memory resources. 
{{pytest: cfme/tests/infrastructure/test_host_memory.py cfme/tests/infrastructure/test_host_cpu_resources.py --use-provider rhos_uc}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 2331b35b6cce3e2dcf9e5f78962f0f32538c51b0:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
No tests were collected in PRT

File: cfme/tests/infrastructure/test_host_memory.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+def pytest_generate_tests(metafunc):
+    argnames, argvalues, idlist = \
+        testgen.infra_providers(metafunc, required_fields=["ssh_credentials"])
+    testgen.parametrize(metafunc, argnames, argvalues, ids=idlist,
+                        scope="module")
+
+
+def get_integer_value(x):
```
Comment (psav) [inline cfme/tests/infrastructure/test_host_memory.py]: @mfalesni did you say you had fixed this?

**Comment by dajoRH** [comment]
Lint report for commit ac057adf19ae99ee0f2a75285bf6261fb40f1e6e:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 44bb789f405938f474aad6b66a7ef35b598262d8:

Everything seems all right from lint perspective :smile: .

Commit inspection report:
**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 3e817bf448b82843911dc50f42d120654e96232b:

Everything seems all right from lint perspective :smile: .

Commit inspection report:
**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 046ae26480245efeda5b77c660d4ca46f0665770:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Comment by rrasouli** [comment]
@dajoRH please merge

**Comment by dajoRH** [comment]
Response to comment #249540194:

It seems like all the basic requirements were met, I have therefore merged your PR.

**Pull request merged**